.github/lm-eval-configs/full-large-models.txt

```diff
@@ -0,0 +1,9 @@
+Meta-Llama-3-70B-Instruct-FP8.yaml
```

Comment:
Not sure where to put this, but it might be good to have a brief README in this repo with: a sketch of hardware requirements for these models and brief description of the various items in the "yaml". As an example for the latter, what does `num_fewshot` mean?

Reply:
Ill add this in the follow up